### PR TITLE
[terra-search-field]search field issues

### PR DESCRIPTION
**Pull request by gt106551**
### Summary

Implemented fixed port size, styled "focus me" button, added condition for default values, and included aria-label for error messages in `terra-search-field`.
**What was changed:**

1. Implemented a fixed port size of 320x256 px for `terra-search-field`.
2. Added styles for the "focus me" button in `terra-search-field`.
3. Added a condition to announce default values for the "search-field-minimum-length" example.
4. Added the "aria-label" attribute to announce error messages for the "search-field-programmatic-set" example.
**Why it was changed:**

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [x] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

[UXPLATFORM-10148](https://jira2.cerner.com/browse/UXPLATFORM-10148)

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-search-field/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 * Fixed
   * Removed unnecessary aria labels and roles to fix redundant announcement of search group issue.
+  * Fixed port resized 320x256 px for `terra-search-field`.
+  * Added styles for "focus me" button for `terra-search-field`.
```
Comment (saket2403) [inline packages/terra-search-field/CHANGELOG.md:8]: This change was made in example not the component itself
```suggestion
```

File: packages/terra-search-field/src/SearchField.module.scss [hunk 2]
```diff
@@ -177,4 +177,9 @@
       color: var(--terra-search-field-input-focus-color);
     }
   }
+  @media (max-width: 320px), (max-height: 256px) {
+    .search-field {
+         width: var(--terra-search-field-resize-max-width, 14.5rem);
```
Comment (saket2403) [inline packages/terra-search-field/src/SearchField.module.scss:182]: Did design team provide this value?

File: packages/terra-core-docs/CHANGELOG.md [hunk 3]
```diff
@@ -6,8 +6,10 @@
 
 * Added
   * Added a test example for `terra-button-group` to validate focus when none of the buttons are selected.
-  * Added visual labels for `terra-search-field` examples. 
+  * Added visual labels for `terra-search-field` examples.
   * Added doc example for `terra-progress-bar`.
+  * Added condition to announce default values for "search-field-minimum-length" example `terra-search-field`.
+  * Added "aria-label" attribute to announce error message for "search-field-programmatic-set" example for `terra-search-field`.
```
Comment (saket2403) [inline packages/terra-core-docs/CHANGELOG.md:12]: Move this to unreleased section

**Comment by rahulkumar8cs** [comment]
+1 for the changes.

**Review by saket2403 — APPROVED** [review]

**Review by sugan2416 — APPROVED** [review]

File: packages/terra-core-docs/CHANGELOG.md (hunk 3 above)
Comment (gt106551) [reply packages/terra-core-docs/CHANGELOG.md]: Updated.

File: packages/terra-search-field/src/SearchField.module.scss (hunk 2 above)
Comment (gt106551) [reply packages/terra-search-field/src/SearchField.module.scss:182]: yes

File: packages/terra-search-field/CHANGELOG.md (hunk 1 above)
Comment (gt106551) [reply packages/terra-search-field/CHANGELOG.md]: Updated

**Pull request merged**